packages/tools/examples/interpolationContourSegmentation/index.ts

```diff
@@ -0,0 +1,337 @@
+const configuration = {
+  interpolation: { enabled: true },
+};
+interpolationTools.set('FreeformInterpolation', {
+  baseTool: PlanarFreehandContourSegmentationTool.toolName,
+  configuration,
+});
+interpolationTools.set('SplineInterpolation', {
+  baseTool: SplineContourSegmentationTool.toolName,
+  configuration,
+});
+interpolationTools.set('LivewireInterpolation', {
+  baseTool: LivewireContourSegmentationTool.toolName,
+  configuration,
+});
+const interpolationToolName = [...interpolationTools.keys()][0];
+
+const segmentIndexes = [1, 2, 3, 4, 5];
+
+// ======== Set up page ======== //
+setTitleAndDescription(
+  'Contour Freehand Annotation Tool',
+  'Here we demonstrate how to use the Contour Freehand Annotation Tool to draw 2D closed ROIs'
+);
+
+const size = '500px';
+const content = document.getElementById('content');
+const viewportGrid = document.createElement('div');
+
+viewportGrid.style.display = 'flex';
+viewportGrid.style.display = 'flex';
+viewportGrid.style.flexDirection = 'row';
+
+const element1 = document.createElement('div');
+const element2 = document.createElement('div');
+element1.style.width = size;
+element1.style.height = size;
+element2.style.width = size;
+element2.style.height = size;
+
+// Disable right click context menu so we can have right click tool
+element1.oncontextmenu = (e) => e.preventDefault();
+element2.oncontextmenu = (e) => e.preventDefault();
+
+viewportGrid.appendChild(element1);
+viewportGrid.appendChild(element2);
+
+content.appendChild(viewportGrid);
+
+addDropdownToToolbar({
+  labelText: 'Segment Index',
+  options: { values: segmentIndexes, defaultValue: segmentIndexes[0] },
+  onSelectedValueChange: (nameAsStringOrNumber) => {
+    updateActiveSegmentIndex(Number(nameAsStringOrNumber));
+  },
+});
+
+const toolsNames = [
+  ...interpolationTools.keys(),
+  PlanarFreehandROITool.toolName,
+  PlanarFreehandContourSegmentationTool.toolName,
+  SplineContourSegmentationTool.toolName,
+  SplineROITool.toolName,
+  LivewireContourSegmentationTool.toolName,
+  LivewireContourTool.toolName,
+];
+let selectedToolName = toolsNames[0];
+
+addDropdownToToolbar({
+  options: { values: toolsNames, defaultValue: selectedToolName },
+  onSelectedValueChange: (newSelectedToolNameAsStringOrNumber) => {
+    const newSelectedToolName = String(newSelectedToolNameAsStringOrNumber);
+    const toolGroup = ToolGroupManager.getToolGroup(toolGroupId);
+
+    // Set the new tool active
+    toolGroup.setToolActive(newSelectedToolName, {
+      bindings: [
+        {
+          mouseButton: MouseBindings.Primary, // Left Click
+        },
+      ],
+    });
+
+    // Set the old tool passive
+    toolGroup.setToolPassive(selectedToolName);
+
+    selectedToolName = <string>newSelectedToolName;
+  },
+});
+
+addButtonToToolbar({
+  title: 'Accept AutoGenerated',
+  onClick: () => {
+    cornerstoneTools.utilities.contours.interpolation.InterpolationManager.acceptAutoGenerated(
```

Comment:
This should be a much shallower utility something like `cstools.utilities.contours.acceptAutogeneratedInterpolations` and that one can call this dee